src/GuppyDataExplorer/index.jsx

```diff
@@ -91,6 +91,7 @@ class Explorer extends React.Component {
               dropdowns: explorerConfig[this.state.tab].dropdowns,
               terraExportURL: explorerConfig[this.state.tab].terraExportURL,
               terraTemplate: explorerConfig[this.state.tab].terraTemplate,
+              sevenBridgesExportURL: explorerConfig[this.state.tab].sevenBridgesExportURL,
```

Comment:
i'm start think manybe we should check if these optional value exists before enabling corresponding buttons, but might be out of the scope of your ticket 🤔

Reply:
That makes sense -- I'll see if there's a simple way I can add that. What do you think of adding a check when portal reads the config, so portal fails to build if the config has the terra / sbg export buttons but is missing the `sevenBridgesExportURL` / `terraExportURL` parameters

Reply:
fails to build sounds a little bit too harsh to me. Personly I think throwing an error in the console would be enough. Even better if we can disable the button

Reply:
Gotchu. My reasoning with fail to build was that we would catch configuration errors during our PR checks. I'll plan on adding an error check to this code, logging an error to the console, and disabling the button.

Reply:
yeah i'm totally with you on that. We have the sanity-check.js which can bail out if sth is not right. But we also have places only throws errors. I guees both are acceptable, just the first option sounds more work on your plate to me 😛

Reply:
Haha that sounds good. Thanks again for the R!